Algolia.Search.Test/BaseTest.cs

```diff
@@ -5,33 +5,33 @@ namespace Algolia.Search.Test
 {
 	public class BaseTest
 	{
-		public static string _testApplicationID = "";
-		public static string _testApiKey = "";
+		public static string _testApplicationID;
+		public static string _testApiKey;
 
-		public AlgoliaClient _client;
+		public IAlgoliaClient _client;
 
-		public Index _index;
+		public IIndex _index;
 		public Analytics _analytics;
 		public IndexHelper<TestModel> _indexHelper;
 
 		// MCM specific
 		public static string _testApplicationIDMCM = "";
 		public static string _testApiKeyMCM = "";
-		public AlgoliaClient _clientMCM;
+		public IAlgoliaClient _clientMCM;
 		public string _userID;
 
 		public BaseTest()
 		{
 			_testApiKey = Environment.GetEnvironmentVariable("ALGOLIA_API_KEY");
 			_testApplicationID = Environment.GetEnvironmentVariable("ALGOLIA_APPLICATION_ID");
-			_client = new AlgoliaClient(_testApplicationID, _testApiKey);
+            _client = new AlgoliaClient(_testApplicationID, _testApiKey);
```

Comment:
of course this is a stupid tiny nit, but it seems like this is intended with spaces instead of tabs

Reply:
Yeah sorry the world moved away from VS "dynamic spacing" a while back and I had to alter my configs accordingly. The solution lacks team settings so VS will have gone with the defaults on my system.

As the next change will potentially be around ripping out the http client etal and further benchmarking/alterations it will happen again. If algolias going with tabs everywhere feel free to save a .editorconfig file in there

Reply:
I don't think we have a particular opinion here, just noticed it since this PR came up and I noticed it wasn't consistent anymore WRT spacing. Really everything should have prettier :)